starkliup/starkliup

```diff
@@ -37,33 +37,53 @@ if test -t 1; then
 fi
 
 main() {
-    if [ -z "$1" ]; then
-        install
-        echo
-        completions
+    RELEASE_TAG=latest
 
-        echo
-        echo "Installation successfully completed."
-    else
-        case $1 in
+    while [ $# -gt 0 ]; do
+        case "$1" in
             -h|--help)
                 usage
                 exit 0
                 ;;
+
+            -v|--version)
+                if [ $# -gt 1 ]; then
+                    RELEASE_TAG="$2"
+                    shift
+                else
+                    echo "${red}Version tag is missing${normal}" 1>&2
+                    exit 1
+                fi
+                ;;
             *)
                 echo "${red}Unknown option: $1${normal}" 1>&2
                 echo "Run ${yellow}starkliup --help${normal} to see usage." 1>&2
                 exit 1
+                ;;
         esac
-    fi
+        shift
+    done
+
+    install
+    echo
+    completions
+
+    echo
+    echo "Installation successfully completed."
 }
 
 install() {
-    echo "Installing the latest version of starkli..."
+    local release_path
```

Comment:
What is this for? Doesn't seem to be used anywhere.

Reply:
Hum, I am not sure I get this one I don't understand the diff.

The `Installing the latest version of starkli...` is actually used as someone may not provide any tag, which is installing the last version.

Or I missed the point sorry, can you detail?

Reply:
Good catch, thanks you.